packages/store/src/text-adapter.ts

```diff
@@ -181,6 +185,15 @@ export class Text {
       this._yText.meta = { delete: true };
     });
   }
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  cover(index: number, length: number, content: string, attributes?: Object) {
+    this._transact(() => {
+      this._yText.delete(index, length);
+      this._yText.insert(index, content, attributes);
+      // @ts-ignore
```

Comment:
Don't use `@ts-ignore` here, check out what we are doing here https://github.com/toeverything/blocksuite/blob/master/packages/store/src/text-adapter.ts#L92-L107